com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ApplicationCloudModelBuilder.java

```diff
@@ -117,18 +116,10 @@ protected CloudApplicationExtended getApplication(Module module) {
                                                 .dockerInfo(parseParameters(parametersList, new DockerInfoParser()))
                                                 .attributesUpdateStrategy(getApplicationAttributesUpdateStrategy(parametersList))
                                                 .v3Metadata(ApplicationMetadataBuilder.build(deploymentDescriptor, module,
-                                                                                             resourcesAndResourceTypes))
+                                                                                             getApplicationServices(module)))

```

Comment:
What will happen if resource has type "existing-service-key"

Reply:
Tested, seems fine as the filtering works.